docs/UserGuide.md

```diff
@@ -113,6 +112,151 @@ Example:
 
 {Example outcome screenshot}
 
+## 4.1 Services Management
+
+This feature allows you to manage the services that your home-based beauty salon provides. You can record the 
+following information about your services: `TITLE`, `DURATION` and `PRICE`.
+
+### 4.1.1 Service Management Command Parameters
+
+Before you dive into using the feature, you may want to have a look at the common parameter used in this feature.
+The table below shows a list of command parameters that will be used in this feature.
+
+| Parameter Name | Description | Example
+|---------|---------|---------
+|`TITLE`| The title of the service you are providing. It must be alphanumeric words not more than 50 characters long.|E.g `Lash Lift`
+|`DURATION`| The duration of the service in hours. <br> <br> It be in half hour intervals.| E.g `1.5`
+|`PRICE`| The revenue received from the service. <br> <br> It must be in dollars.| E.g `5.50`
+
+#### 4.1.2 Add a new service: `addsvc`
+
+You can use this command to add a new service GrAB3.
+
+Format: `addsvc t/TITLE d/DURATION p/PRICE`
+
```

Comment:
I think we need a additional section below every "Format" to tell the user to reference the command parameters. For example:
ℹ️ | Refer to Section 4.1.1, “Service Management Command Parameters” for more detail about each parameter.
-- | --

Reply:
And replicate this 'success' for the other parts till 4.1.7 😄

Reply:
Can refer to @galvinleow 's PR for this ^.